ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-510] Add capital which savings screen

## Description of change
Adds ‘What savings does your client have inside or outside the UK?’ screen and adds validation 

## Link to relevant ticket
[CRIMAPP-510](https://dsdmoj.atlassian.net/browse/CRIMAPP-510)

## Notes for reviewer

NOTE: this is a preliminary PR which does not handle submission, check your answers page etc.. as this sets up some initial capital controllers, routes that will be reused by other devs working on the capital screens. I will follow up with a PR to manage the submission

## Screenshots of changes (if applicable)

### Before changes:

### After changes:

<img width="1159" alt="Screenshot 2024-02-16 at 17 43 19" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/56a14975-6811-4c4b-bcde-a481a7f41bb2">
<img width="1159" alt="Screenshot 2024-02-16 at 17 50 54" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/b4ec62e0-b28d-4c7e-912d-e8d7bc685191">

## How to manually test the feature
Pull down this branch
Start a application 
Go to `/steps/capital/which_savings_does_client_have`
Click save and continue and check the validation error is in place 
Select an option and select save and continue and confirm the validation error is not shown 
There isn't another page linked up so for now it will just refresh the page

[CRIMAPP-510]: https://dsdmoj.atlassian.net/browse/CRIMAPP-510?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
